routes/view.js

```diff
@@ -338,19 +336,19 @@ function formatPPIs(ppis, user_id, callback){
 	for (var i = 0; i < ppis.length; i++){
 		// Parse interaction and create unique ID
 		var ppi   = ppis[i],
-			ppiName = [ppi.source, ppi.target].sort().join("*");
+			ppiName = [ppi.source_id, ppi.target_id].sort().join("*");
```

Comment:
These are now foreign keys to a gene table and the convention Django uses is to append _id to the relation name.